tests/test_answers.py

```diff
@@ -0,0 +1,42 @@
+"""All tests for posting answers to questions"""
+
+import unittest
+import json
+
+from app import create_app
+
+
+class TestQuestions(unittest.TestCase):
```

Comment:
This class should be renamed something representative of what it does. I.e. `AnswersTests`.

Reply:
My bad! Class name has been updated.